### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by judytabaca**

### CODE REVIEW

File: tests/unit/utils/toHoursAndMinutes.spec.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    expect(output).toBeTruthy;
+  });
+
+  it("returns string", () => {
+    expect(typeof output).toBe("string");
+  });
+
+  it("return string in correct format", () => {
+    const re = /\dh\s\d{2}min/;
+    expect(output).toMatch(re);
```
Comment (rossinek) [inline tests/unit/utils/toHoursAndMinutes.spec.js]: tbh. it's much easier to understand the function itself that the test - it should be the other way around 😉

File: tests/unit/utils/toHoursAndMinutes.spec.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import toHoursAndMinutes from "@/utils/toHoursAndMinutes";
+
+const input = 71;
+const output = toHoursAndMinutes(input);
+
+describe("toHoursAndMinutes", () => {
```
Comment (rossinek) [inline tests/unit/utils/toHoursAndMinutes.spec.js:3]: 1. let's see what the outline of this test looks like:
```
toHoursAndMinutes
  - returns value ✅
  - returns string ✅
  - return string in correct format ✅
```

does it really tell you something about this function? what are the edge cases here?

some suggestion for another outline
```
toHoursAndMinutes
  - returns __(?)__ when number is less than or equal to 0 ✅
  - returns time in minutes when number is less than 60 ❌
  - return time in hours and minutes when number is greater than or equal to 60 ✅
```

2. lets see how particular test looks like:
```js
it("return string in correct format", () => {
    const re = /\dh\s\d{2}min/;
    expect( output ).toMatch( re );
})
```
and compare it to this:
```js
it("return string in correct format", () => {
    expect( toHoursAndMinutes(71) ).toMatch( "1h 11min" );
})
```
which test tells you something about how this function actually works?

File: src/utils/toHoursAndMinutes.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const toHoursAndMinutes = (timeInMinutes) => {
+  const hours = Math.floor(timeInMinutes / 60);
+  const minutes = timeInMinutes % 60;
+
+  if (minutes < 10) {
+    return `${hours}h 0${minutes}min`;
```
Comment (rossinek) [inline src/utils/toHoursAndMinutes.js]: I'm not sure if a leading 0 is necessary. it seems quite unnatural to see e.g. `1h 06min` 🤔

**Review by rossinek — COMMENTED** [review]
Great first steps! 👩‍🔬  Technically it is very good, I added some general suggestions

File: tests/unit/utils/toHoursAndMinutes.spec.js (hunk 1 above)
Comment (judytabaca) [reply tests/unit/utils/toHoursAndMinutes.spec.js]: when I step back and look at this again now that youve pointed it out - I really need to be thinking from the third's person perspective when writing tests 😅 NOTED 🗒️ ✔️

File: tests/unit/utils/toHoursAndMinutes.spec.js (hunk 2 above)
Comment (judytabaca) [reply tests/unit/utils/toHoursAndMinutes.spec.js:3]: thank you for this!!! I really appreciate the essay!!! 💚  I'll make a point of making the tests more descriptive from functioning point of view and testing actual functionality and edge cases :)

File: tests/unit/utils/toHoursAndMinutes.spec.js [hunk 4]
```diff
@@ -1,19 +1,15 @@
...
-  it("returns string", () => {
-    expect(typeof output).toBe("string");
+  it("returns time in minutes when number is less than 60", () => {
+    expect(toHoursAndMinutes(59)).toMatch("59min");
   });
 
   it("return string in correct format", () => {
-    const re = /\dh\s\d{2}min/;
-    expect(output).toMatch(re);
+    expect(toHoursAndMinutes(71)).toMatch("1h 11min");
```
Comment (rossinek) [inline tests/unit/utils/toHoursAndMinutes.spec.js:13]: I suggest to add one more expect to this test just to make sure:
```js
expect(toHoursAndMinutes(60)).toMatch( ? );
expect(toHoursAndMinutes(71)).toMatch("1h 11min");
```

Comment (judytabaca) [reply tests/unit/utils/toHoursAndMinutes.spec.js:13]: thank you!!! 👀

Comment (rossinek) [reply tests/unit/utils/toHoursAndMinutes.spec.js:13]: precisely speaking, it was a suggestion to check how your function works exactly for `60`, because (after reading the tests) I have the impression that this is not what is expected 😉

Comment (rossinek) [reply tests/unit/utils/toHoursAndMinutes.spec.js:13]: @judytabaca see above comment (that's the last piece to finish it) :)

Comment (judytabaca) [reply tests/unit/utils/toHoursAndMinutes.spec.js:13]: Artur my biggest apologies, i've went through the task again yesterday - fingers crossed this is looking better now :)

**Review by rossinek — APPROVED** [review]

File: tests/unit/utils/toHoursAndMinutes.spec.js (hunk 4 above)
Comment (rossinek) [reply tests/unit/utils/toHoursAndMinutes.spec.js:13]: Nothing to apologize for, great job I just wanted it to be merged already! 🚀
